src/sql/src/plan/statement/ddl.rs

```diff
@@ -4214,6 +4246,46 @@ pub fn plan_alter_object_rename(
     }
 }
 
+pub fn plan_alter_cluster_rename(
+    scx: &StatementContext,
+    object_type: ObjectType,
+    name: Ident,
+    to_name: Ident,
+    if_exists: bool,
+) -> Result<Plan, PlanError> {
+    match resolve_cluster(scx, &name, if_exists)? {
+        Some(entry) => Ok(Plan::AlterClusterRename(AlterClusterRenamePlan {
+            id: entry.id(),
+            name: entry.name().to_string(),
+            to_name: to_name.into_string(),
```

Comment:
I just realized, I think we need to normalize this name like we do for replica names. Unless there's a reason that I'm unaware of.

Reply:
`into_string` and `normalize::ident` do the same thing. It's subtle though and `normalize::ident` would be better. Also `normalize::ident` should use the new `into_string` function` instead of `as_str().to_string()`.

Reply:
Ah got it. It might even be better to just move this call up a level into `plan_alter_object_rename` so they're all handled the same and it might help avoid potential confusion.